pages/qci_page.py

```diff
@@ -25,30 +25,32 @@ def header(self):
     #######################
     # QCI Spinner Methods #
     #######################
-    def build_qci_spinner_xpath(self, text=None):
+    def build_qci_spinner_xpath(self, text=None, spin_class=None):
```

Comment:
You could set the default value for spin_class='spinner-md' since you're confirming that some non-null value is used in the xpath string. Since it's an optional argument, it would be safe to assume that someone explicitly passing None has a good reason